src/client/Microsoft.Identity.Client.Broker/RuntimeBroker.cs

```diff
@@ -371,6 +372,15 @@ public async Task RemoveAccountAsync(ApplicationConfiguration appConfig, IAccoun
                         }
                         else
                         {
+                            _logger.Info("[WamBroker] Processing WAM exception");
+                            _logger.Verbose($"[WamBroker] TelemetryData: {result.TelemetryData}");
+
+                            string errorMessage =
```

Comment:
Isn't the Error object the same across all APIs ? Can this logic (log all error details + exception message should also contain all error details). be applied everywhere?

Reply:
Can this be merged into WamAdapters.HandleResponse or WamAdapters.ThrowExceptionFromWamError?